benchmark/synthetic/models/lstm_text_generation_nietzsche.py

```diff
@@ -0,0 +1,182 @@
+"""Benchmark a lstm model on Nietzsche dataset
```

Comment:
consider combining these files - `benchmark/synthetic/models/lstm_text_generation_nietzsche.py` & `benchmark/synthetic/models/lstm_text_generation_wikitext2.py` - & use an array with the dataset file names to iterate over their benchmark results. Or use a helper file - to avoid redundant code in both the benchmarking scripts